### PR TITLE
Add Nav

### LINKED ISSUE #21: Button on Homepage
We need a lil arrow 

Screen Shot 2020-07-16 at 11 06 07 AM

### PR DESCRIPTION
**Pull request by chrismekim**
Designs: 

Desktop: https://share.getcloudapp.com/KouB69qg

Mobile: https://share.getcloudapp.com/geuweEz0

Closes #21

### CODE REVIEW

**Comment by brendonsanctuary** [comment]
The fixed title top left 
Screen Shot 2020-07-13 at 1 34 54 PM
 "Studio Carbon Negative" Should still be visible above the menu. Right now its underneath the menu.

**Comment by brendonsanctuary** [comment]
Have menu background fade from 95% opacity to 0% gradient. So you can still see a lil content behind the menu . 95% opacity pictured below. 

Screen Shot 2020-07-13 at 1 35 56 PM

**Comment by brendonsanctuary** [comment]
The menu doesn't display anymore.

**Comment by brendonsanctuary** [comment]
The main image on the home page seems to have farther down and more to the right than before.

Feature Peek
Screen Shot 2020-07-14 at 11 26 52 AM


Where it should sit

Screen Shot 2020-07-14 at 11 27 57 AM

**Comment by brendonsanctuary** [comment]
Reconfigure gradient so it's smooth.
Possible solution?
Screen Shot 2020-07-14 at 3 15 49 PM

**Comment by brendonsanctuary** [comment]
On the three pages. "Buying Offsets" "Helping Clients" & "Calculating Emissions"

The first caption gets moved to the footer when you open the menu.

Screen Shot 2020-07-14 at 3 17 09 PM

**Comment by brendonsanctuary** [comment]
Animation Detail on the menu. 

1. Gradient Background slides in and "Menu" fades out and "Close" fades in
2. After a a little time "Table of Contents" and  all the pages including the button should fade in and move from the top down.

File: src/clients.html [hunk 1]
```diff
@@ -11,11 +11,58 @@
...
+      <div class="nav-menu-button-container fixed z-overlay flex items-center col-12 xl:col-6 xl:off-6 justify-between">
+        <div class="table-of-contents opacity-0 transition-shorter col-6 text-menu-md serif">
+          Table of Contents
+        </div>
+        <button id="nav-menu-button" class="col-6 flex justify-end button--style-nav text-left italic text-menu-md serif transition-shorter">
+          Menu
+        </button>
+        <button id="nav-close-button" class="button--style-nav text-left flex justify-end italic text-menu-md serif none opacity-0 events-none transition-shorter">
+          Close
+        </button>
```
Comment (limsohee1002) [inline src/clients.html:31]: We need aria-label for bottons

File: src/clients.html [hunk 2]
```diff
@@ -11,11 +11,58 @@
...
+                <button class="button--style-nav text-left menu-link opacity-0"><a href="emissions.html">Calculating Emissions</a</button>
+              </div>
+              <div>
+                <button class="button--style-nav text-left menu-link opacity-0"><a href="offsetting.html">Buying Offsets</a</button>
+              </div>
+              <div>
+                <button class="button--style-nav text-left menu-link opacity-0"><a href="clients.html">Helping Clients</a</button>
+              </div>
+              <div>
+                <button class="button--style-nav text-left menu-link opacity-0"><a href="resources.html">Resources</a></button>
```
Comment (limsohee1002) [inline src/clients.html:51]: If you are using a tag you don't have to wrap that with button tag.
We need aria-label for buttons

File: src/clients.html [hunk 3]
```diff
@@ -11,11 +11,58 @@
...
+              </div>
+            </div>
+            <div class="xl:col-6 xl:off-6">
+              <a class="button--style-primary flex items-center justify-center color-black mt1 md:mt2 menu-link opacity-0" href="https://medium.com/sanctuary-computer-inc/our-open-source-2019-profit-loss-12af726471a5">&rarr; See Our 2019 Report</a>
```
Comment (limsohee1002) [inline src/clients.html:55]: aria-label missing

File: src/emissions.html [hunk 4]
```diff
@@ -10,12 +10,59 @@
...
+      <div class="nav-menu-button-container fixed z-overlay flex items-center col-12 xl:col-6 xl:off-6 justify-between">
+        <div class="table-of-contents opacity-0 transition-shorter col-6 text-menu-md serif">
+          Table of Contents
+        </div>
+        <button id="nav-menu-button" class="col-6 flex justify-end button--style-nav text-left italic text-menu-md serif transition-shorter">
+          Menu
+        </button>
+        <button id="nav-close-button" class="button--style-nav text-left flex justify-end italic text-menu-md serif none opacity-0 events-none transition-shorter">
+          Close
+        </button>
```
Comment (limsohee1002) [inline src/emissions.html:31]: aria-label missing for buttons

**Comment by limsohee1002** [comment]
Is there any way to reuse the menu component? so that you don't have to add same code each file?

**Comment by brendonsanctuary** [comment]
Adding the issues over here.


Caption not up against the image on Calculating Emissions.
Screen Shot 2020-07-16 at 11 06 07 AM


Homepage button. 
Screen Shot 2020-07-16 at 11 25 00 AM

**Comment by brendonsanctuary** [comment]
Homepage image should be more centered vertically and spaced more evenly between the words "Studio" & "Carbon"

68747470733a2f2f63646e2e666561747572657065656b2e636f6d2f67656e657269632f636135313533646530633431333436312f5261636b4d756c74697061727432303230303731302d312d317966346f6e342e706e67

**Comment by brendonsanctuary** [comment]
Mobile captions don't look like they are following the .8 lineheight.
Screen Shot 2020-07-16 at 12 02 20 PM

File: src/css/settings/_transitions.scss [hunk 5]
```diff
@@ -24,3 +24,27 @@ $transition: $transition-duration-short $transition-easing;
...
+  0% {
+    opacity: 1;
+    transform: translate3d(0, 0, 0);
+  }
+
+  50% {
+    opacity: 0;
+    transform: translate3d(0, -1rem, 0);
+  }
+}
```
Comment (mokaymokay) [inline src/css/settings/_transitions.scss:50]: maybe these should be animation-fade-in-down and animation-fade-out-up?

Comment (chrismekim) [reply src/css/settings/_transitions.scss]: Changed!

Parcel bundler requires .html but when the site is launched we will remove these!

**Pull request merged**